crates/bevy_asset/src/server/mod.rs

```diff
@@ -433,7 +433,10 @@ impl AssetServer {
         });
 
         #[cfg(not(any(target_arch = "wasm32", not(feature = "multi_threaded"))))]
-        infos.pending_tasks.insert(handle.id(), task);
+        {
+            let mut infos = infos;
+            infos.pending_tasks.insert(handle.id(), task);
+        }
```

Comment:
This is unrelated to the PR. I added this to suppress a Clippy warning around unused `mut`.